### DISCUSSION TITLE
Vote for Preferred Mockup Design

### DISCUSSION THREAD

**Discussion by smbv** [uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp#115] [Polls]
**Description:**
Each member of the web management team has created unique mockups for the website design in task https://github.com/uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp/issues/43. Now, it's time for everyone to vote for their favorite one. The mockup with the most votes will be chosen for implementation. Feel free to provide constructive feedback in your comments.

**Steps:**
1. Review the mockups linked below
2. Consider the usability, aesthetics, and alignment with project objectives.
3. Vote for your preferred mockup.

Mockups:
Option 1 - @smbv: https://github.com/uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp/issues/43#issuecomment-1955871281
Option 2 - @christianperez42: https://github.com/uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp/issues/43#issuecomment-1958075731
Option 3 - @Adriana-Isabel: https://github.com/uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp/issues/43#issuecomment-1959943250
Option 4 - @GiovanniMaysonet: https://github.com/uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp/issues/43#issuecomment-1975396902

**Comment by rukitoyordan** [comment]
@smbv your Mockup is the winner!! I will close the discussion since the deadline has passed.
